internal/controller/pipelinerollout_controller.go

```diff
@@ -568,9 +570,12 @@ func pipelineIsUpdating(pipelineRollout *apiv1.PipelineRollout, newPipelineDef *
 	}
 
 	// note if Pipeline's children are still being updated
-	_, pipelineChildResourceReason := getPipelineChildResourceHealth(pipelineRollout.Status.Conditions)
+	unhealthyOrProgressing, _ := checkChildResources(existingPipelineStatus.Conditions, func(c metav1.Condition) bool {
+		// TODO: once we have "inProgressStrategy" we can just look for c.Status == metav1.ConditionFalse while inProgressStrategy==PPND
```

Comment:
is there a numaflow issue for this? can we add it here in the comment?

Reply:
no longer a numaflow issue but rather just a numaplane issue which I can reference in the comment: https://github.com/numaproj/numaplane/issues/239